docs/_docs/windows.md

```diff
@@ -154,30 +38,18 @@ source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
 ```
 
- * **Note:** We use an unsecure connection because SSL throws exceptions in the version of Ruby
  * Open a command prompt, target your local blog repository root, and install github-pages: `bundle install`
 
+After this process you should have github-pages installed on your system and you can host your blog with `jekyll s`.
```

Comment:
Can we recommend the "long-form" of the command here as well, please?
```
...and you can host your blog with `jekyll serve` or simply `jekyll s`.
```

Reply:
Sounds good - I'll change this.